themes/dulcie/dulcie.theme.sh

```diff
@@ -3,18 +3,14 @@
 # Simplistic one-liner theme to display source control management info beside
 # the ordinary Linux bash prompt.
 #
-# Demo:
-#
-# [ritola@localhost ~]$ cd .bash-it/themes/dulcie
-# [ritola@localhost |master ✓| dulcie]$ # This is single line mode
-# |bash-it|± master ✓|
-# [ritola@localhost dulcie]$ # In multi line, the SCM info is in the separate line
-#
 # Configuration. Change these by adding them in your .bash_profile
 
 DULCIE_COLOR=${DULCIE_COLOR:=1} # 0 = monochrome, 1 = colorful
 DULCIE_MULTILINE=${DULCIE_MULTILINE:=1} # 0 = Single line, 1 = SCM in separate line
 
+# Configuration for Python/Conda virtual environments
+OMB_PROMPT_SHOW_PYTHON_VENV=${OMB_PROMPT_SHOW_PYTHON_VENV:=true} # Keep this for consistency
```

Comment:
```suggestion
OMB_PROMPT_SHOW_PYTHON_VENV=${OMB_PROMPT_SHOW_PYTHON_VENV:=false} # Keep this for consistency
```

This line is usually put to keep the old behavior for existing users. We haven't been showing the Python environment information in the prompt, so we should keep the default behavior to be not showing it and let the user *opt-in* the Python environment information by setting `OMB_PROMPT_SHOW_PYTHON_VENV` in `~/.bashrc` explicitly.

Reply:
fixed! now the default is `false`.